perl/lib/NeedRestart/UI.pm

```diff
@@ -142,10 +143,11 @@ sub _progress_out {
 
 sub _progress_fin {
    my $self = shift;
+   my $columns;
```

Comment:
The variable should also be set to 80 to prevent perl warnings about using undefined value in line 152.